webapp/src/main/java/org/apache/atlas/web/rest/MigrationREST.java

```diff
@@ -325,6 +326,31 @@ public List<AtlasEntity> searchForType(@PathParam("typeName") String typeName, @
         }
     }
 
+    @POST
+    @Path("update-unique-qualified-name")
+    @Timed
```

Comment:
Remove this annotation

Reply:
We may need to monitor time taken for tenants where workflow is taking more time.